src/components/StyleGuide/Typography.mdx

```diff
@@ -201,32 +604,72 @@ following font family assignment:
   borderColor="ui.border.default"
   borderRadius="5px"
   mb="m"
-  p="s"
+  p="inset.default"
 >
-  <Text>system-ui, sans-serif</Text>
+  <Heading size="heading2" border="0" mb="xs">
+    Lorem ipsum dolor
+  </Heading>
+  <Text size="subtitle1" m="0">
+    Subtitle nullam id dolor id nibh ultricies vehicula ut id elit.
+  </Text>
 </Box>
 }
 
-The following CSS variables are available through the DS.
+#### Subtitle 2
 
-<Source
-  code={`
---nypl-fonts-heading: system-ui, sans-serif;
---nypl-fonts-body: system-ui, sans-serif;
-`}
-  language="css"
-/>
+{
 
-> **RECOMMENDATIONS**  
-> When making font selections for project based components outside the DS,
-> sans serif fonts display better on computers and mobile devices. Avoid
-> decorative or overly stylized fonts, as they are often difficult to read even
-> for users without visual impairments or reading Standard Text Default
+<List type="ul">
+  <li>
+    This style SHOULD be used with text set as a subtitle grouped with a heading
+    element.
+  </li>
+  <li>Text set in this style MAY be sentence length.</li>
+</List>
+}
+
+|                 | JS Token   | CSS Token                          | Value          |
+| --------------- | :--------- | :--------------------------------- | :------------- |
+| **Font Size**   | `text.tag` | `var(--nypl-fontSizes-text-tag)`   | `1rem` (16px)  |
+| **Font Weight** | `text.tag` | `var(--nypl-fontWeights-text-tag)` | `500` (medium) |
+| **Line Height** | --         | --                                 | `1.3`          |
+
+{
+
+<Box
+  border="1px"
+  borderColor="ui.border.default"
+  borderRadius="5px"
+  mb="m"
+  p="inset.default"
+>
+  <Heading size="heading5" border="0" mb="xxs">
+    Lorem ipsum dolor
+  </Heading>
+  <Text size="subtitle2" m="0">
+    Subtitle nullam id dolor id nibh ultricies vehicula ut id elit.
+  </Text>
+</Box>
+}
+
+#### Overline 1
+
+{
 
-## Font Color
+<List type="ul">
+  <li>
+    This style SHOULD be used with text set as a pre-title grouped with a
+    heading element.
+  </li>
+  <li>Text set in this style SHOULD be one or two words in length.</li>
+</List>
+}
 
-All text within a project, unless specifically noted in mockups, should be
-black on a white background.
+|                 | JS Token   | CSS Token                          | Value            |
+| --------------- | :--------- | :--------------------------------- | :--------------- |
+| **Font Size**   | `text.tag` | `var(--nypl-fontSizes-text-tag)`   | `0.75rem` (12px) |
```

Comment:
overline1 and overline2 tables have the same JS token and CSS token values in the table.